### ISSUE TITLE
[Featuere] Create not updateable chunks (and maybe other elements) by option

### ISSUE THREAD

**Issue by Jako** [theboxer/Git-Package-Management#59]
Maybe possible with a second vehicle that has set xPDOTransport::UPDATE_OBJECT to false.

**Comment by theboxer** [comment]
This is already possible, you can do it with this:

```
,"build": {
  "attributes": {
    "chunks": {
      "update_object": false
    }
  }
}
```

**Comment by Jako** [comment]
Thanks. That should be documented.

But my idea goes a bit further: If I am right, you could have different vehicles in one package and each vehicle could have different „update_object“ settings. That way fixed and updatable chunks (and other objects) could be possible.

Tell me, if I am wrong with that.

**Comment by theboxer** [comment]
Yes you're right, but in GPM all elements are transported in same vehicle and currently there's no option to do it differently...
